Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    hocon (1.0.0)
+    hocon (1.0.1)
```

Comment:
Why is a .lock file included at all? Seems like this will just cause problems for contributors.